executor/errors.go

```diff
@@ -48,6 +48,7 @@ var (
 	ErrDBaccessDenied              = terror.ClassExecutor.New(mysql.ErrDBaccessDenied, mysql.MySQLErrName[mysql.ErrDBaccessDenied])
 	ErrTableaccessDenied           = terror.ClassExecutor.New(mysql.ErrTableaccessDenied, mysql.MySQLErrName[mysql.ErrTableaccessDenied])
 	ErrBadDB                       = terror.ClassExecutor.New(mysql.ErrBadDB, mysql.MySQLErrName[mysql.ErrBadDB])
+	ErrTableIsNotView              = terror.ClassExecutor.New(mysql.ErrWrongObject, "'%s.%s' is not VIEW")
```

Comment:
should we use the error messages from `mysql.MySQLErrName`?

Reply:
This error code may be a little different, `mysql.ErrWrongObject` has different message, witch is `xxx is not view` and `xxx is not base table`, and `mysql.MySQLErrName` didn't contain these messages.

Reply:
In MySQL:
```sql
MySQL(root@localhost:test) > show create view t;
ERROR 1347 (HY000): 'test.t' is not VIEW
```

Notice that the error code is `1347`, after searching in MySQL code base, the error pattern is:
```
{ "ER_WRONG_OBJECT", 1347, "\'%-.192s.%-.192s\' is not %s","HY000", "", 316 },
```

So in https://github.com/pingcap/tidb/pull/9309/files#diff-df919d9361195b93310b219738ea1aeeR764, I think the error we should use is:
```go
ErrWrongObject.GenWithStackByArgs(db.Name.O, tb.Meta().Name.O, "VIEW")
```

Reply:
You are right, I will fix all this kind error handle.